ownCloudAppShared/User Interface/StaticTableView/StaticTableViewRow.swift

```diff
@@ -444,10 +444,6 @@ open class StaticTableViewRow : NSObject, UITextFieldDelegate {
 			cellTextField.bottomAnchor.constraint(equalTo: (cell?.contentView.bottomAnchor)!, constant:-14).isActive = true
 		}
 
-		self.updateViewFromValue = { [weak cellTextField] (row) in
```

Comment:
This would remove the ability to update the contents of text fields by setting the `value` property of `StaticTableViewRow`s.

The problem seems to be that:
- entering text into the textfield calls `textFieldContentChanged()`
- `textFieldContentChanged()` sets `.value` to the entered text
- setting `value` invokes `.updateViewFromValue()`, which sets the textfield's string to the entered text, which seems to cause the issue

By making sure that `.updateViewFromValue()` is not called when `.value` is updated via `textFieldContentChanged()`, this should be possible to avoid. I've quickly put together a proposal on how that could be achieved in branch `fix/japanese-input-support-alternative` in commit https://github.com/owncloud/ios-app/commit/34176217c8b6426aaed96dd99b29a2443157ba9b (can be merged directly into this PR if you find it to work (I didn't test it) and want to use it).

Reply:
@felix-schwarz yes, you are right. Just used your alternative. Thanks for your proposal!